requirements.txt

```diff
@@ -1,69 +1,44 @@
-alabaster==0.7.10
```

Comment:
I just noticed one missing package: ``setup.py`` requires ``argparse``, which we apparently need for some scripts in ``tools/``. Should we add pin a specific version in requirements.txt as well?

Reply:
imho yes please

Reply:
It looks like `argparse` is part of the python base distribution, so we don't need to add it explicitly.

Reply:
You're right, I missed this. Should we then just remove it from ``setup.py``?

Reply:
on it.

Reply:
fixed in d4ae1646